server/index.js

```diff
@@ -1,14 +1,20 @@
 const express = require('express');
-var bodyParser = require('body-parser');
+
 let app = express();
-app.use(express.static(__dirname + '/../client/dist'));
+var bodyParser = require('body-parser');
+let retrieveOne = require('../database/seed.js').retrieveOne;
 
-app.post('/', function (req, res) {
-  res.status(201).send();
-});
+app.use(bodyParser.json());
+app.use(express.static(__dirname + '/../client/dist'));
 
-app.get('/', function (req, res) {
-  res.status(200).send('testing');
+app.get('/shop/:productId', (req, res) => {
+  let item = req.params.productId;
+  retrieveOne(item, (err, data) => {
+    if (err) {
+      res.status(404).send('error');
```

Comment:
you don't need to send `'error'`, that is what `.status(404)` is doing.  You do need to add the line `.end()`.
Something like this: `res.status(404).end();`